### PR TITLE
Address CVE-2022-42003

### PR DESCRIPTION
**Pull request by kaushalmahi12**
**Is your feature request related to a problem? Please provide an existing Issue # , or describe.**
Address [issue](https://github.com/opensearch-project/performance-analyzer/issues/290) in backport of 2.x.

**Describe the solution you are proposing**

**Describe alternatives you've considered**

**Additional context**

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/performance-analyzer/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by kkhatua — DISMISSED** [review]

**Comment by kkhatua** [comment]
@kaushalmahi12 Does this need changes to RCA package as well???

File: build.gradle [hunk 1]
```diff
@@ -240,7 +240,7 @@ checkstyleTest.enabled = false
 dependencies {
 
     def jacksonVersion = "2.13.4"
-    def jacksonDataBindVersion = "2.13.4"
+    def jacksonDataBindVersion = "2.13.4.2"
```
Comment (sgup432) [inline build.gradle:243]: Need this change in RCA as well - https://github.com/opensearch-project/performance-analyzer-rca/blob/main/build.gradle#L319

**Comment by kaushalmahi12** [comment]
> @kaushalmahi12 Does this need changes to RCA package as well???

Raised the [RCA PR](https://github.com/opensearch-project/performance-analyzer-rca/pull/243) to upgrade the dependecies.

**Review by dzane17 — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (kkhatua) [reply build.gradle:243]: Done in https://github.com/opensearch-project/performance-analyzer-rca/pull/243

**Review by kkhatua — APPROVED** [review]

**Pull request merged**